enigma-swing/src/main/java/org/quiltmc/enigma/gui/GuiController.java

```diff
@@ -167,8 +168,15 @@ public CompletableFuture<Void> openMappings(MappingFormat format, Path path) {
 				this.refreshClasses();
 				this.chp.invalidateJavadoc();
 				this.statsGenerator = new StatsGenerator(this.project);
+				new Thread(() -> {
```

Comment:
This is already inside another secondary thread, and I feel like we have a better way to spawn threads instead of `new Thread()`

Reply:
Is it executor time?

Reply:
I have no idea what the best way of doing this would be, this is kinda a bodge that just got left in because I don't know what's better